src/app/input-container/input-container.component.css

```diff
@@ -0,0 +1,16 @@
+.container {
+  width: 100%;
+  height: 100vh;
```

Comment:
If the intention was to center the component's content, then it is not working - I have scrollbars, when open it in a browser (Chrome 79)